docs/_source/community/developer_docs.md

```diff
@@ -158,18 +165,21 @@ source .env/bin/activate
 Then, you just need to install Argilla with the command below. Note that we will install it in editable mode using the -e/--editable flag in the `pip` command to avoid having to re-install it on every code modification, but if you’re not planning to modify the code, you can just omit the -e/--editable flag.
 
 ```sh
+cd argilla
```

Comment:
Do we really need to add `cd argilla` in this and upcoming blocks? The first block after "Install with `pip`" already covers this step.